verticapy/connect.py

```diff
@@ -392,7 +392,14 @@ def read_dsn(section: str, dsn: str = ""):
     if confparser.has_section(section):
 
         options = confparser.items(section)
-        conn_info = {"port": 5433, "user": "dbadmin", "session_label": "verticapy"}
+        conn_info = {
+            "port": 5433,
+            "user": "dbadmin",
+            "session_label": "verticapy-"
+            + verticapy.__version__
+            + "-"
+            + str(verticapy.options["identifier"]),
```

Comment:
Vertica generates a unique SESSION_ID each time a new connection is made. IMHO, that is the most meaningful ID if you need an identifier to log the activities related to a specific VerticaPy connection.

Reply:
Session ID is unique but sometimes the user uses multiple queries with different sessions. We need an idea to merge them together. A verticapy session is different than a Vertica session.

Reply:
I don't understand your claim: "A verticapy session is different than a Vertica session."? What do you mean by verticapy session?

Reply:
When a user import verticapy. He can uses multiple connection to Vertica for the same import. We need to track that.

Reply:
Does each VerticaPy connection correspond to a VerticaPy session, or the group of all connections following the same `import verticapy`?